src/plugin.py

```diff
@@ -30,6 +30,16 @@
 
 
 class BNetPlugin(Plugin):
+    async def get_game_time(self, game_id, context):
+        qpTimeMinutes = 0
+        log.info('game_id: ' + game_id)
+        if game_id == "5272175": # Overwatch
+            player_data = await self.backend_client.get_ow_player_data()
+            qpTime = re.search('0x0860000000000026.+?(?:(?P<h>\d+):)?(?P<m>\d+)', player_data.text)
+            qpTimeMinutes = int(qpTime.group('h')) * 60 + int(qpTime.group('m'))
+            log.info('Overwatch quickplay playtime [minutes]: ' + str(qpTimeMinutes))
+        return GameTime(game_id, qpTimeMinutes, None)
```

Comment:
You can easilty add competitive playtime using the same `data-stat-id="0x0860000000000026"` like this:

```suggestion
        if game_id == "5272175":  # Overwatch
            game_time_minutes = 0
            player_data = await self.backend_client.get_ow_player_data()
            matches = re.findall('0x0860000000000026.+?(?:(?P<h>\d+):)?(?P<m>\d+)', player_data.text)
            for match in matches:
                game_time_minutes += int(match[0]) * 60 + int(match[1])
            return GameTime(game_id, game_time_minutes, None)

        return GameTime(game_id, None, None)
```

Reply:
Theoretically yes. But we found out that they only show the stats for the current competitive season. There's no good way to properly fetch a total play time of all ranked seasons. :(

Reply:
Oh, you are right. That's too bad :(

Reply:
is it true also for https://ow-api.com/ (only one season)?

Reply:
@elkangaroo changes also fixes returning time for other titles: `None` should be returned for unknown value.

Reply:
> is it true also for https://ow-api.com/ (only one season)?
yes, unfortunately it's the same.